src/Illuminate/Database/DatabaseManager.php

```diff
@@ -16,6 +17,8 @@
  */
 class DatabaseManager implements ConnectionResolverInterface
 {
+    use Macroable;
```

Comment:
```suggestion
    use Macroable {
        __call as macroCall;
    }
```

Reply:
As @taylorotwell suggests,  classes that already have a __call method need some additional logic to make Macroable work. You also need to update the __call() method accordingly,




`
__call($method, $parameters) { 
return static::hasMacro($method)
                    ? $this->macroCall($method, $parameters)
                    : $this->connection()->$method(...$parameters); }
`

Reply:
@sulemaanhamza Thx...!  🤓 I was hoping to have time to look into finishing this up this weekend and it looks like you've already done all the hard work.  😜 I definitely appreciate it.

I'll take some time later today or tomorrow to look more closely at your suggestion and try to get this thing wrapped up.  Thanks again.  🚀